sources/packages/web/src/views/aest/student/FileUploads.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <v-card class="mt-4">
+    <div class="mx-5 py-4">
+      <content-group>
+        <v-row class="m-2">
+          <v-col class="category-header-medium color-blue">File Uploads</v-col>
+        </v-row>
+        <DataTable
+          :value="studentFileUploads"
+          :paginator="true"
+          :rows="DEFAULT_PAGE_LIMIT"
+          :rowsPerPageOptions="PAGINATION_LIST"
+        >
+          <template #empty>
+            <p class="text-center font-weight-bold">No records found.</p>
+          </template>
+          <Column
+            field="groupName"
+            header="Document Purpose"
+            sortable="true"
+          ></Column>
+          <Column field="metadata" header="Application #">
+            <template #body="slotProps"
+              ><span v-if="slotProps.data.metadata">{{
+                `${slotProps.data.metadata.applicationNumber}`
+              }}</span
+              ><span v-else>----</span></template
```

Comment:
why `----`?

Reply:
Single hyphen was not looking good, so used ---- 😄 , devs can suggest anything to use here.

Reply:
I believe you need to follow Figma. or ask for Lynn suggestion, IMO

Reply:
Sure @ann-aot  I will ask lynn, thanks for the suggestion